src/content/docs/en/guides/content-collections.mdx

```diff
@@ -391,7 +391,7 @@ You can use this to filter by any content criteria you like. For example, you ca
 // Example: Filter content entries with `draft: true`
 import { getCollection } from 'astro:content';
 const draftBlogEntries = await getCollection('blog', ({ data }) => {
-  return data.draft !== true;
+  return data.draft === true;
```

Comment:
Since filtering is probably mostly used to NOT include drafts, let's revert this to: 

```suggestion
  return data.draft !== true;
```
But update the copy in the code comment. (Stupid GitHub won't let me select all these lines together in one comment, so that one'll come next.)

Reply:
Hey thanks @sarah11918! We'll also need to change the variable name, as otherwise the conflict remains. I assume `blogEntries` is good enough?

Reply:
D'oh! Yup, you're right! Got a good variable name for NON-draft posts? maybe something like publishedPosts ?

(The example is so much easier to write for drafts, but most people are going to want NOT drafts!  :sweat_smile:  So, I think we should optimize for code people can use. We'll jump on the grenade and make it nice!)

Reply:
Oh sorry, I made an edit there and we crossed wires. I'm fairly easy here, `publishedPosts` or `publishedBlogEntries` both seem fine, do you prefer the former?

Reply:
No preference! We'll go with yours!

Reply:
I think I *do* like the explicitness of `published` though, rather than just `blogEntries`, for illustrating what we're doing. 

So, my slight vote is for `publishedBlogEntries` over `blogEntries`. What do you think? Am I being toooo pedantic?